docs/integrations/sources/intercom.md

```diff
@@ -43,9 +42,13 @@ The Intercom connector should not run into Intercom API limitations under normal
 
 ## Getting started
```

Comment:
this should have a cloud and OSS section, see salesforce docs for an examplee

Reply:
Updated.